packages/frontend-2/components/header/NavUserMenu.vue

```diff
@@ -143,23 +143,24 @@ import {
   ChatBubbleLeftRightIcon
 } from '@heroicons/vue/24/outline'
 import { Roles } from '@speckle/shared'
-import type { Optional } from '@speckle/shared'
 import { useActiveUser } from '~~/lib/auth/composables/activeUser'
 import { useAuthManager } from '~~/lib/auth/composables/auth'
-import { loginRoute } from '~~/lib/common/helpers/route'
 import { useTheme, AppTheme } from '~~/lib/core/composables/theme'
 import { useServerInfo } from '~/lib/core/composables/server'
+import type { RouteLocation } from 'vue-router'
+
+defineProps<{
+  loginUrl?: RouteLocation & { href: string }
```

Comment:
What's the point requiring this prop to be so complicated if you only use it as a bool in the template? It could easily just be `shouldShowSignIn: boolean`

Reply:
It's also used as the 'to' prop in the NuxtLink

`            <NuxtLink
              :class="[
                active ? 'bg-foundation-focus' : '',
                'flex gap-3.5 items-center px-3 py-2.5 text-sm text-primary cursor-pointer transition'
              ]"
              :to="loginUrl"
            >
              <ArrowRightOnRectangleIcon class="w-5 h-5" />
              Sign In
            </NuxtLink>`

Reply:
@andrewwallacespeckle Right, my bad. In that case, it still doesn't necessarily have to be RouteLocation it can be `RouteLocationRaw`, which also supports basic `strings` and thus is way less complicated to feed in

Reply:
<img width="249" alt="image" src="https://github.com/specklesystems/speckle-server/assets/938316/41320ef6-3b50-4b9f-b378-be2861b560a4">
<img width="634" alt="image" src="https://github.com/specklesystems/speckle-server/assets/938316/6be258a9-75f0-49c9-a571-8a230562b916">